apphost/test/Test.cs

```diff
@@ -0,0 +1,8 @@
+﻿Console.WriteLine("Hello, World!");
+#pragma warning disable CS8600
```

Comment:
Please leave a comment here explaining what this piece of code is testing for (i.e. the relation to the signal handler)